lua/user_example/init.lua

```diff
@@ -216,16 +201,59 @@ local config = {
 
   mappings = {
     -- first key is the mode
+    -- desc setting is stored by vim.keymap.set() as a part of opts table in vim lua module
     n = {
       -- second key is the lefthand side of the map
-      ["<C-s>"] = { ":w!<cr>", desc = "Save File" },
+      -- Terminal
+      -- Install ncdu package: (Disk size w/o snapshot btrfs subvolumes and .git)
+      ["<leader>tu"] = { function() astronvim.toggle_term_cmd "ncdu -x --exclude .git" end, desc = "Term for NCDU w/o junks" },
+      -- Install mc package:
+      ["<leader>tm"] = { function() astronvim.toggle_term_cmd "mc . ~" end, desc = "Term for MC" },
+      -- Install libreply-perl + libterm-readline-gnu-perl packages:
+      ["<leader>tp"] = { function() astronvim.toggle_term_cmd "reply" end, desc = "Term for perl5 REPL" },
```

Comment:
could we remove thse and leave the `tu` example for a lua function example?

Reply:
Yes